Upstream-Status: Pending [Not submitted to upstream yet]
Signed-off-by: Arunachalam Ganapathy <arunachalam.ganapathy@arm.com>

From 1827607c6381339d051a36a84d749c3a50e3463d Mon Sep 17 00:00:00 2001
From: Arunachalam Ganapathy <arunachalam.ganapathy@arm.com>
Date: Mon, 16 Nov 2020 17:38:07 +0000
Subject: [PATCH 5/5] plat-totalcompute: define tzdram start address for S-EL1
 SPMC config

Define TZDRAM_START for CFG_CORE_SEL1_SPMC config

Change-Id: I71aaa92a0ecdb791802336ca31ab52e97ee52955
Signed-off-by: Arunachalam Ganapathy <arunachalam.ganapathy@arm.com>
---
 core/arch/arm/plat-totalcompute/conf.mk | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/core/arch/arm/plat-totalcompute/conf.mk b/core/arch/arm/plat-totalcompute/conf.mk
index 34095acf..d330eb92 100644
--- a/core/arch/arm/plat-totalcompute/conf.mk
+++ b/core/arch/arm/plat-totalcompute/conf.mk
@@ -24,7 +24,10 @@ endif
 ifeq ($(PLATFORM_FLAVOR),tc0)
 CFG_TEE_CORE_NB_CORE = 4
 
-ifeq ($(CFG_CORE_SEL2_SPMC),y)
+ifeq ($(CFG_CORE_SEL1_SPMC),y)
+CFG_TZDRAM_START ?= 0xfd000000
+CFG_TZDRAM_SIZE  ?= 0x02000000
+else ifeq ($(CFG_CORE_SEL2_SPMC),y)
 CFG_TZDRAM_START ?= 0xfd281000
 CFG_TZDRAM_SIZE  ?= 0x01d7f000
 else
-- 
2.26.2

